BedrockServer.cpp

```diff
@@ -1671,11 +1671,14 @@ void BedrockServer::_status(unique_ptr<BedrockCommand>& command) {
         {
             // Make it known if anything is known to cause crashes.
             shared_lock<decltype(_crashCommandMutex)> lock(_crashCommandMutex);
+            vector<string> crashCommandList;
             size_t totalCount = 0;
             for (const auto& s : _crashCommands) {
+                crashCommandList.push_back(s.first);
```

Comment:
I would do something like:
```
crashCommandList.push_back(s.first + ":" + to_string(s.second.size()));
```

Reply:
Hmm `to_string(s.second.size())`, this would save the count of each occurrence, right. Not sure if that would be useful. 
I asked on slack [here](https://expensify.slack.com/archives/C094TGUTZ/p1731009701954069?thread_ts=1730930618.630709&cid=C094TGUTZ) if saving the parameters would be useful and ill try to save that instead. I can save counts of each additionally if people feel like it would be useful. Atm i don't know if it's useful, especially if we print each failure instance.

So I'm thinking of this code,
```
for (const auto& s : _crashCommands) {
    totalCount += s.second.size();
    for (const STable& params : s.second) {
        crashCommandList.push_back(s.first + ":" + SComposeJSONObject(params));
    }
}
```
To print the command and its params each time. (oh actually i should print s.first just once and then print the params as an array, ill do that next)

For testing i tried setting crash command like this,
```
CRASH_COMMAND
Content-Length:16

DeleteReport

CRASH_COMMAND
Content-Length: 30

GuidedSetup
accountID: 12345

CRASH_COMMAND
Content-Length: 30

GuidedSetup
accountID: 67890
```

and i get an output that looks like this,
```
{"commandCount":1,"commandPortBlockReasons":[],"CommitCount":201201,"crashCommandList":["DeleteReport:{}","GuidedSetup:{\"accountID\":12345}","GuidedSetup:{\"accountID\":67890}"],"crashCommands":3,"host":"0.0.0.0:4445","isLeader":true,"multiWriteEnabled":true,"multiWriteManualBlacklist":[],"peerList":[],"plugins":[{"name":"Auth","version":1},{"name":"DB"},{"name":"expensifyBackupManager","version":"44cf19575a"}],"priority":-1,"queuedCommandList":[],"state":"LEADING","syncNodeAvailable":true,"syncThreadQueuedCommandList":[],"version":"228614ce9e:Auth_1:expensifyBackupManager_44cf19575a"}
```
which looks nice with params but i don't know why the accountID key in crashCommandList is escaped. Any suggestions on how i can improve it? (I'll consolidate s.first so that GuidedSetup, shows up just once)

Reply:
> i don't know why the accountID key in crashCommandList is escaped

I think because the whole thing is inside quotes. It kinda looks like invalid json - maybe not technically, but at least not how I would expect. I would expect this:

```json
[
    "DeleteReport:{}",
    "GuidedSetup:{\"accountID\":12345}",
    "GuidedSetup:{\"accountID\":67890}"
]
```

to be more like this perhaps:

```json
[
    {"DeleteReport": {}},
    {"GuidedSetup": {"accountID":12345}},
    {"GuidedSetup":{"accountID":67890}}
]
```

Reply:
Could you use a JSON object? There are lots of good examples in Auth - Tyler, any particular reason the JSON libraries are there as opposed to libstuff?

Reply:
Updated the code to show params. I've added examples in the description. It kinda looks like this,
```
[
    {
        "DeleteReport": []
    },
    {
        "GuidedSetup": [
            {"accountID": 12345},
            {"accountID": 67890}
        ]
    }
]
```
Is that good or would you guys like to see it any different?

Reply:
That format looks great to me 👍